### DISCUSSION TITLE
[2021/12/14] Frontend Night

### DISCUSSION THREAD

**Discussion by jjenko** [wantedly/frontend-night#183] [Archive]
## Policy

- 資料の準備をがんばらない
- 参加・途中退席は自由
- スキップしない

話したい・聞きたいネタを書いてく ✏️

ハッシュタグ [#frontend_night](https://twitter.com/hashtag/frontend_night?src=hash)

Prev: https://github.com/wantedly/frontend-night/discussions/182 

 _Created from https://github.com/wantedly/frontend-night/discussions/179 by [issue-creator](https://github.com/rerost/issue-creator)_

**Comment by kobayang** [comment]
React Conf の感想をみんなでワイワイしたい

https://conf.reactjs.org/stage

**Comment by hayaokimura** [comment]
後で読む: Dan 先生が react team で学んだ 100 のこと
https://twitter.com/dan_abramov/status/1470613731071696896?s=12

**Comment by chloe463** [comment]
## [型推論のTips集](https://qiita.com/wintyo/items/6ff67816f718a8fe6398?utm_source=pocket_mylist)

filter の結果の型から null を除きたいやつは地味に便利

**Reply by kobayang** [reply]
filter の例が便利だった
